parts/linux/cloud-init/artifacts/aks-check-network.sh

```diff
@@ -100,7 +100,7 @@ function check_and_curl {
         if [ $response -ge 200 ] && [ $response -lt 400 ]; then
             logs_to_events "AKS.testingTraffic.success" "echo '$(date) - SUCCESS: Successfully tested $url with returned status code $response'"
             break
-        elif [ $response -eq 400 ] && ([ $url == "acs-mirror.azureedge.net" ] || [ $url == "eastus.data.mcr.microsoft.com" ]); then
```

Comment:
So strange that shellcheck doesn't pickup that $url should be in quotes - if there is a space in the url then weird things will happen.

Reply:
I just looked into that. It was because it was a `info` SC2086 not an error and 4 years ago this `info` was set to ignore.
Let me enable it and fix related errors in the next PR otherwise this PR will be very huge.
Thanks for catching this though. It reduces the chance of potential errors.

Reply:
I'm mostly aware about this at the moment because my IDE underlies all unquoted shell variables and warns me about them. Which is handy, but annoying.